doc/classes/EditorTranslationParserPlugin.xml

```diff
@@ -54,19 +54,19 @@
 		}
 		[/csharp]
 		[/codeblocks]
-		To add a translatable string associated with a context, plural, or comment:
+		To add a translatable string associated with a context, plural, comment, or source line:
 		[codeblocks]
 		[gdscript]
-		# This will add a message with msgid "Test 1", msgctxt "context", msgid_plural "test 1 plurals", and comment "test 1 comment".
-		ret.append(PackedStringArray(["Test 1", "context", "test 1 plurals", "test 1 comment"]))
+		# This will add a message with msgid "Test 1", msgctxt "context", msgid_plural "test 1 plurals", comment "test 1 comment", and source line "7".
```

Comment:
Because this comment grew in length, it's not going to be an easy read, not that it was before. This will not look pretty in the built-in and online class reference.
I would suggest removing it, but at the same time, that would be inconsistent with the other comments below, which are... of questionable utility, too.